changelogs/fragments/574-qos-wred-yellow-red-support.yaml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - sonic_qos_wred - Update QoS WRED module for yellow and red colors (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/574).
```

Since an existing choice is removed, please change the type to 'breaking_changes'
```suggestion
breaking_changes:
  - sonic_qos_wred - Add support for yellow and red colors (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/574).
```